README.md

```diff
@@ -28,3 +28,23 @@ CHROMEDRIVER_PATH=/path/to/chromedriver bundle exec rake
 
 If you do not have chromedriver installed, first execute
 `bin/install-chromedriver`, which will install it for you and report the path.
+
+## Webpacker
+
+We do not use the Rails asset pipeline: we use
+[Webpacker](https://github.com/rails/webpacker) for everything. Webpacker
+sometimes doesn't give clear indications of what's wrong when it doesn't work.
+
+**If you have trouble getting assets to compile in dev**, a good place to start
+debugging is by running `bin/webpack`. This will run the webpacker compilation
```

Comment:
it'd be good to provide one link to the documentation

Reply:
There is a link two lines up — do you think it ought to be made more explicit?